### PR TITLE
Forms Automated Test - LPS-133473 | master 

### PR DESCRIPTION
**Pull request by fmarcio**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase [hunk 1]
```diff
@@ -398,14 +398,83 @@ definition {
...
+		Form.closeSidebar();
 
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Select from List");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 1",
+			optionValue = "1");
+
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase:427]: @fmarcio is it possible to use `FormFields.editOptionsRow` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase [hunk 2]
```diff
@@ -398,14 +398,83 @@ definition {
...
+		FormFields.addSingleSelectionOption(
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase:428]: @fmarcio is it possible to use `FormFields.editOptionsRow` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase [hunk 3]
```diff
@@ -398,14 +398,83 @@ definition {
...
 	test PreserveTheMultipleSelectionsProperty {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133473 PreserveTheMultipleSelectionsProperty pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Form.gotoElementSetsTab();
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase:410]: @fmarcio Can you please use `Navigator.gotoNavItem(navItem = "Element Sets");` instead to keep consistency? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase [hunk 4]
```diff
@@ -398,14 +398,83 @@ definition {
...
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.selectSelectOption(
+			fieldName = "Select from List",
+			selectOption = "Option 1");
+		
+		FormFields.enableCheckbox(fieldName = "Option 2");
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase:472]: @fmarcio is it possible to use `FormFields.selectSelectOption` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase (hunk 1 above)
Comment (fmarcio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase]: > @fmarcio is it possible to use `FormFields.editOptionsRow` instead?

Fixed!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase (hunk 2 above)
Comment (fmarcio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase]: > @fmarcio is it possible to use `FormFields.editOptionsRow` instead?

@carolmariaabb  Fixed!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase (hunk 3 above)
Comment (fmarcio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase]: > @fmarcio Can you please use `Navigator.gotoNavItem(navItem = "Element Sets");` instead to keep consistency? Thanks!

@carolmariaabb  Fixed! :-)

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**